charts/values.yaml

```diff
@@ -74,7 +74,7 @@ preUpgradeHook:
     # -- The container image registry URL for the hook job
     registry: docker.io
     # -- The container repository for the hook job
-    repo: bitnamilegacy/kubectl
+    repo: openebs/kubectl
```

Comment:
oh I thought I had done this already, must have missed this one!
I think we need same fix on this repo: <https://github.com/openebs/mayastor-extensions/commit/544a3f14041543736adc1fe0948908377e50ed59>
Please make that change and see if it would catch this

Reply:
@tiagolobocastro This change is already there in mayastor-extensions.